package.json

```
@@ -1118,56 +1118,94 @@
     "walkthroughs": [
       {
         "id": "confluent-walkthrough",
-        "title": "Connect to Kafka Clusters",
+        "title": "Get Started with Confluent for VS Code",
         "description": "Interact with Apache Kafka® clusters running locally or on Confluent Cloud.",
         "steps": [
           {
-            "id": "connect-local-cluster",
-            "title": "Connect to Kafka",
-            "description": "**Confluent Cloud Clusters** \nLog in to your Confluent account to see your Confluent Cloud resources. \n[Log in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)\n**Local Clusters** \nIf you have Kafka running locally, it appears automatically in the Resources view. \n[Start Local Resources](command:confluent.docker.startLocalResources)",
+            "id": "connect-conflulent-cloud",
+            "title": "Connect to Confluent Cloud",
+            "description": "Log in to see your Confluent Cloud resources.\n[Log in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)",
```

Minor suggestion since we're here anyway and all other parts of the extension use "Sign In":
```suggestion
            "description": "Sign in to see your Confluent Cloud resources.\n[Sign in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)",
```

---

Updated